requirements.txt

```diff
@@ -8,11 +9,13 @@ django-rest-framework==0.1.0
 djangorestframework==3.14.0
 djongo==1.3.6
 dnspython==2.2.1
+jsonschema==4.17.1
 mypy-extensions==0.4.3
 pathspec==0.10.2
 platformdirs==2.5.4
 pycodestyle==2.9.1
 pymongo==3.12.3
+pyrsistent==0.19.2
```

Comment:
what's this ?

Reply:
It's one of the dependencies of the `jsonschema` package.